service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1617,20 +1619,24 @@ components:
 
     HintQuery:
       type: object
-      description: Get display hints
+      description: |
+        Get display hints. Only one of relatedEntity and entityQuery may be provided
```

Comment:
It would be good to describe what passing a query actual does here since it's quite different from the other behavior. I don't think you can add it to the `entityQuery` parameter because it's a ref but even better there if you actually can.

Reply:
It cannot be added to the individual ref property . adding here